website/forms/php/shortcodes.md

```diff
@@ -0,0 +1,24 @@
+---
+id: shortcodes
+title: Shortcodes
+---
+
+## What are shortcodes?
+
+Shortcodes are small pieces of code that you can use to add predefined forms to your website. They are used to simplify the process of adding forms to your website.
+
+## esFormsLink
+
+This shortcode is used to add a simple link to your form. For example you need to add a privacy policy link to your form checkbox label.
+
+```php
+[esFormsLink url="https://eightshift.com" label="Eightshift" /]
+```
```

Comment:
~~~suggestion
```
[esFormsLink url="https://eightshift.com" label="Eightshift" /]
```
~~~

@goranalkovic-infinum you can use `~~~` instead of backticks when you want to wrap something that has a backtics in the suggestions.